java/test/org/openqa/selenium/grid/config/JsonConfigTest.java

```diff
@@ -32,4 +34,18 @@ void shouldUseATableAsASection() {
 
     assertThat(config.get("cheeses", "selected")).isEqualTo(Optional.of("brie"));
   }
+
+  @Test
+  void ensureCanReadListOfMaps() {
+    String raw = "{\"node\":{\"detect-drivers\":false,"
+      + "\"driver-configuration\":[{\"display-name\":\"htmlunit\","
+      + "\"stereotype\":{\"browserName\":\"htmlunit\",\"browserVersion\":\"chrome\"}}]}}";
+    Config config = new JsonConfig(new StringReader(raw));
+    List<String> expected = Arrays.asList(
```

Comment:
The expected value should be:
```java
List<String> expected = Arrays.asList(
    "display-name=\"htmlunit\"",
    "stereotype={\"browserName\"=\"htmlunit\", \"browserVersion\"=\"chrome\"}"
);
```
The output of this method is used by **NodeOptions**, so the values need to be represented as JSON.